test/run.html

```diff
@@ -10,5 +10,6 @@
 	<script type="text/javascript" src="../node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
 	<script type="text/javascript" src="../test/run-browser.js" ></script>
 	<script type="text/javascript" src="../montage.js" data-module="./all"></script>
+	<!-- <script type="text/javascript" src="../montage.js" data-module="./auth-specs.js"></script> -->
```

Comment:
Any reason to leave that comment?

Reply:
Nope! Removed. https://github.com/montagejs/montage/pull/1984/commits/39e7b828333474a4fc664667a8c6e91e88fac7a8